src/Python/sirf/contrib/BSREM/BSREM.py

```diff
@@ -48,7 +57,8 @@ def step_size(self):
     def update(self):
         g = self.subset_gradient(self.x, self.subset)
         self.x_update = (self.x + self.eps) * g / self.average_sensitivity * self.step_size()
-        self.FOV_filter.apply(self.x_update)
+        if self.iteration_filter:
```

Comment:
I think it would be clearer if this was:
```suggestion
        if self.iteration_filter is not None:
```